packages/vehicle-rental-overlay/src/DefaultMarkers/index.js

```diff
@@ -0,0 +1,146 @@
+import { divIcon } from "leaflet";
+import memoize from "lodash.memoize";
+import { stationType } from "@opentripplanner/core-utils/lib/types";
+import PropTypes from "prop-types";
+import React from "react";
+import ReactDOMServer from "react-dom/server";
+import { CircleMarker, Marker } from "react-leaflet";
+
+import { floatingBikeIcon, hubIcons } from "../bike-icons";
+import * as Styled from "../styled";
+
+/**
+ * This file contains default marker types for rental vehicles,
+ * that can be used when defining the VehicleRentalOverlay's symbol prop:
+ * - Circle
+ * - GenericMarker
+ * - HubAndFloatingBike
+ */
+
+// Prop types reused across components.
+const templatePropTypes = {
+  /** The children of the component. */
+  children: PropTypes.node,
+  /** The rental vehicle or station to render. */
+  entity: stationType.isRequired
+};
+const templateDefaultProps = {
+  children: null
+};
+
+/**
+ * Helper function to create a Circle component to render entities
+ * using a fixed size, fill color, and stroke color.
+ * Usage: Circle({ fillColor: "#FF56BB", pixels: 4, strokeColor: "#000000" })
+ */
+export const Circle = ({ fillColor = "gray", pixels, strokeColor }) => {
+  const newStrokeColor = strokeColor || fillColor;
+
+  const GeneratedMarker = ({ children, entity: station }) => (
+    <CircleMarker
+      center={[station.y, station.x]}
+      color={newStrokeColor}
+      fillColor={fillColor}
+      fillOpacity={1}
+      radius={pixels}
+      weight={1}
+    >
+      {children}
+    </CircleMarker>
+  );
+
+  GeneratedMarker.propTypes = templatePropTypes;
+  GeneratedMarker.defaultProps = templateDefaultProps;
+  return GeneratedMarker;
+};
```

Comment:
This marker is only used in the story, so I think it ought to be moved out of here and into the story file instead.

Reply:
Addressed in https://github.com/opentripplanner/otp-ui/pull/159/commits/dcc3b3943c44cffeb34507e9127e8c3b9190098a.